lib/app.jsx

```diff
@@ -89,6 +89,7 @@ function mapDispatchToProps(dispatch, { noteBucket }) {
     setAuthorized: () => dispatch(reduxActions.auth.setAuthorized()),
     setSearchFocus: () =>
       dispatch(actionCreators.setSearchFocus({ searchFocus: true })),
+    toggleMonospace: () => dispatch(settingsActions.toggleMonospace()),
```

Comment:
you can update skip this and add `toggleMonospace` to the `pick(settingsActions)` array.